src/Closure.h

```diff
@@ -47,22 +47,21 @@ class Closure : public IRVisitor {
         Type type;
 
         /** The dimensionality of the buffer. */
-        uint8_t dimensions;
+        uint8_t dimensions{0};
```

Comment:
I thought we typically used: member = 0 vs member{0} for PODs

Reply:
Apparently the latter is what clang-tidy prefers? I can check to see if there is a way to change that.

Reply:
Yeah, there is a way, but I'll just need to re-do this PR in order to avoid a weird history.

Reply:
PTAL